website/source/docs/config/index.html.md

```diff
@@ -197,6 +197,17 @@ For Consul, the following options are supported:
 
   * `scheme` (optional) - "http" or "https" for talking to Consul.
 
+  * `check_timeout` (optional) - The check interval used to send health check
+    information to Consul.  Defaults to "5s".
+
+  * `disable_registration` (optional) - If true, then Vault will not register
+    itself with Vault.  If the Consul Agent for Vault and the Consul Servers
+    are older than `0.6.4` it is required to set this to "true" due to API
+    incompatibilities.  Defaults to "false".
```

Comment:
re: first point: there isn't any builtin way to do this, no.  We could finger print the Consul server and then change our behavior based on the support of the Consul Servers and agent, but that functionality doesn't exist out of the box anywhere atm.

re: point #2, the worst that would happen is a double registration.

Let's touch base re: this today to hash out next steps on these points (I wrestled with them before hitting enter).